biomeasure.py

```diff
@@ -1,29 +1,104 @@
 import pandas as pd
 import numpy
-
+import math
 from bioinput import Bioinput
+import sys
 
-class Biomeasure():
-    """
 
-    """
+class Biomeasure:
+    """ This is the main calling class
 
-    def __init__(self, threshold=0.9):
-        self.__hr_rawdata = pd.DataFrame(numpy.empty((10, 3))*numpy.nan, columns=['HeartRate', 'B/T', 'time'])
+    __init__ sets the __hr_rawdata
+
+    """
+    def __init__(self, threshold=0.8, thr_brady=50, thr_tachy=140):
         self.__threshold = threshold
-        input = Bioinput()
-        self.ecg_file = input.file
+        self.__thr_brady = thr_brady
+        self.__thr_tachy = thr_tachy
+        inputfile = Bioinput()
+        self.ecg_file = inputfile.file
+        self.__hr_rawdata = inputfile.read_input()
```

Comment:
Given how this is being used, you could make `read_input()` part of the `Bioinput.__init__()`, and have the read data be an attribute of the `Bioinput` object and then have that be a subobject of your `Biomeasure` object.

Reply:
Did it!